modules/space/rendering/renderablestars.cpp

```diff
@@ -120,6 +120,61 @@ namespace {
         "to its color. The texture is used as a one dimensional lookup function."
     };
 
+    constexpr openspace::properties::Property::PropertyInfo MappingBvInfo = {
+        "MappingBV",
+        "Mapping (bv-color)",
+        "The name of the variable in the speck file that is used as the b-v color "
+        "variable"
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo MappingLuminanceInfo = {
+        "MappingLuminance",
+        "Mapping (luminance)",
+        "The name of the variable in the speck file that is used as the luminance "
+        "variable"
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo MappingAbsMagnitudeInfo = {
+        "MappingAbsMagnitude",
+        "Mapping (absolute Magnitude)",
+        "The name of the variable in the speck file that is used as the absolute "
+        "magnitude variable"
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo MappingAppMagnitudeInfo = {
+        "MappingAppMagnitude",
+        "Mapping (apparent Magnitude)",
+        "The name of the variable in the speck file that is used as the apparent "
+        "magnitude variable"
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo MappingVxInfo = {
+        "MappingVx",
+        "Mapping (vx)",
+        "The name of the variable in the speck file that is used as the star velocity in "
```

Comment:
extra "in", for all the velocity directions. Should be either "... velocity along the x-axis" or "... velocity in the x direction"